### PR TITLE
Neat 447 rules transformer dms compliant characters

### PR DESCRIPTION
**Pull request by nikokaoja**

### CODE REVIEW

File: cognite/neat/rules/transformers/_converters.py [hunk 1]
```diff
@@ -61,7 +66,100 @@ def _transform(self, rules: T_VerifiedInRules) -> T_VerifiedOutRules:
         raise NotImplementedError()
 
 
-class ToCompliantEntityIds(ConversionTransformer[VerifiedRules, VerifiedRules]): ...
+class ToCompliantEntities(RulesTransformer[InformationInputRules, InformationInputRules]):  # type: ignore[misc]
+    """Converts input rules to rules with compliant entity IDs."""
```
Comment (doctrino) [inline cognite/neat/rules/transformers/_converters.py:70]: Is there some more documentation you can add here? Compliant with what?

Maybe just say which regex pattern you use with the properties

File: cognite/neat/rules/transformers/_converters.py [hunk 2]
```diff
@@ -61,7 +66,100 @@ def _transform(self, rules: T_VerifiedInRules) -> T_VerifiedOutRules:
...
+                suffix=cls._fix_entity(class_.suffix),
+            )
+
+        return class_
+
+    @classmethod
+    def _fix_value_type(
+        cls, value_type: str | DataType | ClassEntity | MultiValueTypeInfo
+    ) -> str | DataType | ClassEntity | MultiValueTypeInfo:
+        fixed_value_type = value_type
```
Comment (doctrino) [inline cognite/neat/rules/transformers/_converters.py:118]: This is not used, you have a value type for each condition in the if-else below

File: cognite/neat/rules/_shared.py [hunk 3]
```diff
@@ -18,6 +18,7 @@
 Rules: TypeAlias = (
     AssetInputRules | DMSInputRules | InformationInputRules | DomainRules | InformationRules | DMSRules | AssetRules
 )
+AllInformationRules: TypeAlias = InformationRules | InformationInputRules
```
Comment (doctrino) [inline cognite/neat/rules/_shared.py:21]: Looks like this is not used, so suggest removing it
```suggestion
```

File: cognite/neat/rules/transformers/_converters.py (hunk 1 above)
Comment (nikokaoja) [reply cognite/neat/rules/transformers/_converters.py:70]: Will add good point

**Pull request merged**